src/rp2_common/pico_unique_id/include/pico/unique_id.h

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (c) 2021 Raspberry Pi (Trading) Ltd.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+#ifndef _PICO_UNIQUE_ID_H_
+#define _PICO_UNIQUE_ID_H_
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/** \file pico/unique_id.h
+ *  \defgroup unique_id unique_id
```

Comment:
should be pico_unique_id

Reply:
also should be added to docs/index.h (I notice that pico_stdio is missing from there, so if you could add that in passing!)

Reply:
pico_stdio seems to be listed fine under the runtime group?

Reply:
apparently I'm blind :)